files/usr/share/terminator/terminatorlib/plugins/host_watch.py

```diff
@@ -250,36 +255,37 @@ def get_last_line(self, terminal):
         we iterate back until LF found (means : end of output of last command), 
         then forward one line
         """
-        lines= vte.get_text_range(start_row, start_col, end_row, end_col, is_interesting_char)
+        lines = vte.get_text_range(start_row, start_col, end_row, end_col, is_interesting_char)[0]
```

Comment:
What happened here?

Reply:
There was an exception raised around the type of `lines` - something about it being a tuple.

I tried finding API docs for the VTE, but ultimately fell back to accessing elements of the tuple until I found the one that worked.